examples/examples.py

```diff
@@ -16,21 +16,22 @@
 os.makedirs("./examples/local_db", exist_ok=True)
 
 availabe_examples = {
+    # small examples for easier understanding of different features and configurations
```

Comment:
Please also change this in the example_simulations.rst file

Reply:
good point, I have now completely reworked that file to include all examples!